### ISSUE TITLE
The stop method blocks the program

### ISSUE THREAD

**Issue by xiaochai1027** [go-echarts/statsview#13]
I'm calling this method 

**ViewManager.Stop()**

_ViewManager.Start()_ also calls the stop method internally

```
for {
		select {
		case <-ticker.C:
			viewer.StartRTCollect()
		case <-vm.done:
			vm.Stop()
			ticker.Stop()
			return
		}
	}
```

**Comment by chenjiandongx** [comment]
@xiaochai1027 Upgrade the statsview version to v0.3.3 as the bug has been fixed at #11

**Comment by xiaochai1027** [comment]
ok，thx
